src/app/styles/font.js

```diff
@@ -0,0 +1,9 @@
+import { Montserrat } from "next/font/google";
```

Comment:
Why do we need local fonts in the public folder in we use next/font/google?

Reply:
My bad) fonts have been removed from public folder. 

A quick question: nextjs recommends using next-font to load the fonts, should we stick with that approach or is it better to store fonts locally in the public folder?

Reply:
It's better to use next/font